frontend/locales/en/common.ts

```diff
@@ -85,6 +85,7 @@ const common = {
     "additional communication info": "additional communication info",
     "communication content": "content of the communication",
     medium: "medium",
+    "by medium": "by",
```

Comment:
is this correct?

Reply:
Yes

Reply:
The "by medium" is confusion, but it is correct in the sentences "by email", "by sms", ...

Reply:
Okay I understand what you mean